src/gui/sdl_mapper.cpp

```diff
@@ -1754,41 +1764,135 @@ static void change_action_text(const char* text,Bit8u col) {
 	bind_but.action->SetColor(col);
 }
 
+static std::string humanize_mod_name(const CBindList &binds, const std::string &fallback)
```

Comment:
Struggling a bit with this function. 
`humanize..` tells me it's making something more readable, but not specifically what.. it could be a short-hand-name to a long-hand name? Or it could be a scan-code number to a button name? 

Ideally we could be more specific in the function name, for example: `scancode_to_key_name`, but if that's not possible, then suggest a nice comment showing a couple examples of what it takes in and spits out.

Reply:
It "humanizes" the modifier key name based on binds associated with the modifier event.

Examples:

- By default, `mod_1` action has two binds associated: `Left Ctrl` and `Right Ctrl` -> this function will turn it into `Ctrl` or `<fallback>: Ctrl` (depending if fallback is empty or not).
- If an event has only a single bind, then name of the bound key is returned intact, e.g. `Right Alt` -> `Right Alt`.
- If user modified binds and added more than 2 keys, or keys that don't have left/right variant, then `<fallback>` is used.

I'll try to come up with a better name.

Reply:
That's a great description! (could be a comment in itself). Tossing this into the option pile: `mod_to_button_name`

Reply:
I simplified the function and used `humanize_key_name` for now.